cartography/intel/gcp/iam.py

```diff
@@ -772,6 +821,12 @@ def attach_role_to_user(
     ON CREATE SET
     pr.firstseen = timestamp()
     SET pr.lastupdated = {gcp_update_tag}
+    WITH user
+    WHERE (NOT EXISTS(user.parent)) OR
+    NOT user.parent IN ['organization', 'folder']
```

Comment:
should we add `project` in the list for the IN check?

Reply:
if we add `project` in this list then for `organization ` level entity parent overwrite by `project`